src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -1117,8 +1117,8 @@ public static String stringifyMultiline(byte[] code) {
 
   static class ByteCodeIterator {
 
-    byte[] code;
-    int pc;
+    public byte[] code;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make code a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWalyDYZdPPRVdwylpM5&open=AWalyDYZdPPRVdwylpM5)

[ISSUE_KEY]: AWalyDYZdPPRVdwylpM5